### DISCUSSION TITLE
Where to place example color and icons configuration

### DISCUSSION THREAD

**Discussion by psi4j** [gokcehan/lf#1023] [Q&A] [answered]
From what I understand, previously LF was set up to have both colors and icons configured in your ~/.*rc file, or something like a .zshenv file. I tried to copy and paste the example configurations as seen in `https://github.com/gokcehan/lf/blob/master/etc/colors.example`, into both my .zshrc and .zshenv and neither one seems to work. When I enable icons in lfrc, I can see the default characters (such as 'd' for dir), but for some reason it doesn't apply the icons to the variables. I was hoping someone could give me a tip as to what I could do to edit the icons and colors, starting with the example file. Any tips would be greatly appreciated.

**Comment by petraglyph** [comment, marked as answer]
You can place config files in `~/.config/lf` without the `.example` to use them (ie `~/.config/lf/lfrc` or `~/.config/lf/icons`)

**Reply by psi4j** [reply]
That makes a whole lot of sense. Thanks again, pennbauman!
